packages/social-metadata-forms/src/SocialMetadataPreviewForm.js

```diff
@@ -183,78 +176,15 @@ class SocialMetadataPreviewForm extends Component {
-		if ( socialMediumName.toString() !== "Twitter" ) {
-			return (
-				<Fragment>
-					<ImageSelectWithCaret
-						label={ imageSelectTitle }
-						onClick={ onSelectImageClick }
-						onRemoveImageClick={ onRemoveImageClick }
-						warnings={ imageWarnings }
-						imageSelected={ imageSelected }
-						onMouseEnter={ this.onImageEnter }
-						onMouseLeave={ this.onLeave }
-						isActive={ activeField === "image" }
-						isHovered={ hoveredField === "image" }
-						imageUrl={ imageUrl }
-						imageAltText={ imageAltText }
-						hasPreview={ ! isPremium }
-						imageUrlInputId={ join( [ lowerCaseSocialMediumName, "url-input", idSuffix ] ) }
-						selectImageButtonId={ join( [ lowerCaseSocialMediumName, "select-button", idSuffix ] ) }
-						replaceImageButtonId={ join( [ lowerCaseSocialMediumName, "replace-button", idSuffix ] ) }
-						removeImageButtonId={ join( [ lowerCaseSocialMediumName, "remove-button", idSuffix ] ) }
-					/>
-					<ReplacementVariableEditor
-						onChange={ onTitleChange }
-						content={ title }
-						placeholder={ titleInputPlaceholder }
-						replacementVariables={ replacementVariables }
-						recommendedReplacementVariables={ recommendedReplacementVariables }
-						type="title"
-						fieldId={ join( [ lowerCaseSocialMediumName, "title-input", idSuffix ] ) }
-						label={ titleEditorTitle }
-						onMouseEnter={ this.onTitleEnter }
-						onMouseLeave={ this.onLeave }
-						onSearchChange={ onReplacementVariableSearchChange }
-						isActive={ activeField === "title" }
-						isHovered={ hoveredField === "title" }
-						withCaret={ true }
-						onFocus={ this.onSelectTitleEditor }
-						onBlur={ this.onDeselectEditor }
-						editorRef={ this.onTitleEditorRef }
-					/>
-					<ReplacementVariableEditor
-						onChange={ onDescriptionChange }
-						content={ description }
-						placeholder={ descriptionInputPlaceholder }
-						replacementVariables={ replacementVariables }
-						recommendedReplacementVariables={ recommendedReplacementVariables }
-						type="description"
-						fieldId={ join( [ lowerCaseSocialMediumName, "description-input", idSuffix ] ) }
-						label={ descEditorTitle }
-						onMouseEnter={ this.onDescriptionEnter }
-						onMouseLeave={ this.onLeave }
-						onSearchChange={ onReplacementVariableSearchChange }
-						isActive={ activeField === "description" }
-						isHovered={ hoveredField === "description" }
-						withCaret={ true }
-						onFocus={ this.onSelectDescriptionEditor }
-						onBlur={ this.onDeselectEditor }
-						editorRef={ this.onDescriptionEditorRef }
-					/>
-					<SettingsNotice>{ __( "To customize the appearance of your post specifically for Twitter, please fill out " +
-						"the 'Twitter appearance' settings below. If you leave these settings untouched, the 'Social appearance' settings " +
-						"mentioned above will also be applied for sharing on Twitter.", "wordpress-seo" ) }</SettingsNotice>
```

Comment:
I moved the rendering of the settings notice to `SocialAppearanceModal` component since there are other logics involved before we can render this notice. 
These logic differs per platform (e.g. between WordPress and Shopify), so it's better handled outside of `social-metadata-forms` package